lms/static/sass/xblocks/image_explorer.scss

```diff
@@ -79,50 +80,54 @@ body.new-theme {
 
         .image-explorer-hotspot {
           text-align: left;
-          background: transparent;
+          background: white;
           border-radius: 22px;
+          color: var(--primary);
+          border-radius: 25px;
```

Comment:
you already defined the border-radius at line number 84, Please remove one.

Reply:
Thanks, @shahidrafique to point out this issue. that issue have been resolved.